playwright/fixtures/fixture-mobile-create.ts

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2025 New Vector Ltd.
+
+SPDX-License-Identifier: AGPL-3.0-only OR LicenseRef-Element-Commercial
+Please see LICENSE in the repository root for full details.
+*/
+
+import { type Browser, type Page, test, expect } from "@playwright/test";
+
+export interface MobileCreateFixtures {
+  asMobile: {
+    creatorPage: Page;
+    inviteLink: string;
+  };
+}
+
+export const mobileTest = test.extend<MobileCreateFixtures>({
+  asMobile: async ({ browser }, puse) => {
+    const fixtures = await createCallAndInvite(browser);
+    await puse({
```

Comment:
```suggestion
  asMobile: async ({ browser }, pUse) => {
    const fixtures = await createCallAndInvite(browser);
    await pUse({
```

So I know how to pronounce it :)

Reply:
`eslint_does_not_want_that_we_call_it_use` ? 🤣